shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -26,6 +32,7 @@ - (instancetype)init {
 
 - (void)dealloc {
   [_lifeCycleDelegate release];
+  [_rootFlutterViewControllerGetter release];
```

Comment:
This class does not create the getter, it seems weird that this class is responsible for destroying it.

Reply:
It actually does create it in its declaration of the property.  This is related to your other comment, too.  The file FlutterAppDelegate_Test is a thing called a category which allows you to extend classes or make methods selectively visible.  You can think of this as declaring that property as `@VisibleForTesting`.